extracters/knoedler_linkedart.py

```diff
@@ -49,15 +49,12 @@ def make_la_book(data: dict):
 	ident = vocab.LocalNumber()
 	ident.content = str(data['identifier'])
 	book.identified_by = ident
-	try:
```

Comment:
What was this trying to catch, and is it not needed anymore? Seems like the `int` cast might still raise an exception...?

Reply:
Yeah, that was the fear, but looking at the data it /is/ always an int.  So if something sneaks in, it probably should raise.